lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -639,9 +639,26 @@ function _dosomething_user_send_to_message_broker() {
     'user_country'      => isset($user_country_code) ? $user_country_code : 'US',
   );
 
-  // 26+ Club: Override Mobile Commons.
+  // Subscribe new members to Do Something Master Campaign.
   $mobile = dosomething_user_get_field('field_mobile', $account);
-  $opt_in = variable_get('dosomething_signup_mobilecommons_opt_in_path_user_register');
+
+  // Load opt-in path for user country.
+  // Note: only US messaging is supported now.
+  // Uncomment this code when more contries are supported.
+  // See https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_signup/dosomething_signup.admin.inc#L43
+
+  /*
+  $user_language = $user->language;
```

Comment:
does this work? the variable is `$account` above. 
also it seems like the country code is above as well from [this line](https://github.com/DoSomething/phoenix/pull/7001/files#diff-ad30a5d569ca2ae6d2e04223aa501104R639) and this call
`list($user_language, $user_country_code) = dosomething_global_user_details($account);
`